databases/week2/create_database.sql

```diff
@@ -0,0 +1,54 @@
+-- In this assignment, I have tried to design a database for recruitment agency who recruit candidate
+--for temporary contract positions different companies in different recruitment categories.
+
+DROP DATABASE IF EXISTS database_week_2;
+CREATE DATABASE IF NOT EXISTS database_week_2;
+USE database_week_2;
+
+CREATE TABLE IF NOT EXISTS recruiter (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `email` varchar(255) NOT NULL,
+  `phone` varchar(255) NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+
+DROP TABLE IF EXISTS company;
+
+CREATE TABLE IF NOT EXISTS company (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `email` varchar(255) NOT NULL,
+  `phone` varchar(255) NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+
+CREATE TABLE IF NOT EXISTS candidate (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `email` varchar(255) NOT NULL,
+  `phone` varchar(255) NOT NULL,
```

Comment:
Wondering to know WHY you want to assign 255 chars for a human name, email, or phone number(they can't be that much longer). Applies to the above too.

Reply:
yes it could be in between 50 to 100 characters.